src/serialization/pkl2proto.py

```diff
@@ -0,0 +1,155 @@
+import pickle
+import sys
+import numpy as np
+import font_pb2
+import serialization 
+
+def parseDict(d, filename= "No filename provided"):
+    '''
+    Read a Dict containing font curves and return the info in a standardized np array
+
+    Parameters
+    ----------
+    d: Python Dictionary with key of the chars and the value of a closed curves dict, 
+    the closed curve dict has keys of ints up to max num closed curves and lists
+    of assosiated bez curves. Each bez curve in the list of bez curves has 3 coordinates
+    which are x and y coords of the control points. 
+    
+    filename: Optional parameter for knowing where the reading of the data failed
+    
+    
+    Returns
+    -------
+    New NP array of size len(chars), maxClosedCurves, maxCurvePerGlyph, 3, 2
+    '''
+
+     chars = [
+        'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N',
+        'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'X', 'Y', 'Z', 'a', 'b', 'c',
+        'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q',
+        'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z', 'zero', 'one', 'two',
+        'three', 'four', 'five', 'six', 'seven', 'eight', 'nine', 'space',
+        'exclam', 'quotedbl', 'numbersign', 'dollar', 'percent', 'ampersand',
+        'quotesingle', 'parenleft', 'parenright', 'asterisk', 'plus', 'comma',
+        'hyphen', 'period', 'slash', 'colon', 'semicolon', 'less', 'equal',
+        'greater', 'question', 'at', 'bracketleft', 'bracketright',
+        'asciicircum', 'underscore', 'grave', 'braceleft', 'bar', 'braceright',
+        'asciitilde'
+    ]
+
+    maxCurvePerGlyph = 80
+    maxClosedCurves = 20
+    
+    # initialize the np array with all ones then multiply by 99
+    final = np.ones((len(chars), maxClosedCurves, maxCurvePerGlyph, 3, 2))
+    final = final * 999 
+    
+    for c in range(len(chars)):
+        try:
+            # this will run per glyph
+
+            # getting number of closed curves in this glyph
+            numClosedCurves = sorted(d[chars[c]].keys())[-1]
```

Comment:
this could just be a max, but idt it matters so much

Reply:
Done.